Apps/WebClient/src/Server/Services/WalletStatusService.cs

```diff
@@ -0,0 +1,116 @@
+        /// <param name="walletDelegate">Injected Wallet delegate.</param>
+        public WalletStatusService(ILogger<WalletStatusService> logger, IWalletDelegate walletDelegate)
+        {
+            this.logger = logger;
+            this.walletDelegate = walletDelegate;
+        }
+
+        /// <inheritdoc />
+        public RequestResult<WalletConnection> UpdateWalletConnection(Guid connectionId)
+        {
+            RequestResult<WalletConnection> result = new ()
+            {
+                ResultStatus = Common.Constants.ResultType.Error,
+            };
+
+            DBResult<WalletConnection> dbResult = this.walletDelegate.GetConnection(connectionId);
+            if (dbResult.Status == DBStatusCode.Read)
+            {
+                WalletConnection connection = dbResult.Payload;
+                result.ResourcePayload = connection;
+                connection.ConnectedDateTime = DateTime.UtcNow;
+                connection.Status = WalletConnectionStatus.Connected;
+                DBResult<WalletConnection> updateResult = this.walletDelegate.UpdateConnection(connection, true);
+                if (updateResult.Status == DBStatusCode.Updated)
+                {
+                    result.ResultStatus = Common.Constants.ResultType.Success;
+                    result.TotalResultCount = 1;
+                }
+                else
+                {
+                    this.logger.LogWarning($"Unable to update wallet connection with id: {connectionId}");
+                    result.ResultError = new RequestResultError() { ResultMessage = "Error Updating Wallet Connection", ErrorCode = ErrorTranslator.ServiceError(ErrorType.CommunicationExternal, ServiceType.WalletIssuer) };
+                }
+            }
+            else
+            {
+                this.logger.LogWarning($"Unable to find wallet connection with id: {connectionId}");
+                result.ResultError = new RequestResultError() { ResultMessage = "Unable to find Wallet Connection", ErrorCode = ErrorTranslator.ServiceError(ErrorType.CommunicationExternal, ServiceType.WalletIssuer) };
+            }
+
+            return result;
+        }
+
+        /// <inheritdoc />
+        public RequestResult<WalletCredential> UpdateWalletCredential(Guid exchangeId)
+        {
+            RequestResult<WalletCredential> result = new ()
+            {
+                ResultStatus = Common.Constants.ResultType.Error,
+            };
+
+            DBResult<WalletCredential> dbResult = this.walletDelegate.GetCredential(exchangeId);
+            if (dbResult.Status == DBStatusCode.Read)
+            {
+                WalletCredential credential = dbResult.Payload;
+                result.ResourcePayload = credential;
+                credential.AddedDateTime = DateTime.UtcNow;
+                credential.Status = WalletCredentialStatus.Added;
+                DBResult<WalletCredential> updateResult = this.walletDelegate.UpdateCredential(credential, true);
```

Comment:
nit, I think the commit is set to true by default

Reply:
Removed as you're correct!